addon/model.js

```diff
@@ -307,17 +255,13 @@ export default class MegamorphicModel extends Ember.Object {
     return this._baseModel != null;
   }
 
-  get _updateStrategy() {
-    // TODO There is no good way to determine which strategy to use, so using some heuristics for now
-    return MERGE_STRATEGY;
```

Comment:
it appears that @hjdivad had the intention of making the update strategy potentially pluggable, should we keep that plumbing?

Reply:
I think he wanted this to be pluggable in Ember Data, not necessarily in M3, but I will let him comment.